examples/angular/src/pages/ui/primitive-poc/flex/flex-demp.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'amplify-flex-demo',
+  templateUrl: 'flex-demo.component.html',
+})
+export class AmplifyFlexExampleComponent implements OnInit {
+  constructor() {}
+
+  ngOnInit() {}
```

Comment:
NIT: let's remove these if they're not getting used 🙏

Reply:
Done.